jre_emul/Classes/java/nio/file/attribute/FileTime.java

```diff
@@ -15,7 +15,7 @@
  *  limitations under the License.
  */
 
-package com.google.j2objc.nio.file.attribute;
+package java.nio.file.attribute;
 
 import java.util.concurrent.TimeUnit;
 
```

Comment:
Please work with Mary to expand the class doc-comment below (GitHub review UI won't let me comment there), to fully explain that this is a complete j2objc rewrite, designed to avoid Android's FileTime java.time dependencies.

Reply:
Done